tfs/writer.py

```diff
@@ -24,17 +25,18 @@
 
 def write_tfs(
     tfs_file_path: pathlib.Path | str,
-    data_frame: TfsDataFrame | pd.DataFrame,
+    data_frame: TfsDataFrame | pd.DataFrame | pd.Series,
     headers_dict: dict | None = None,
     save_index: str | bool = False,  # noqa: FBT002
     colwidth: int = DEFAULT_COLUMN_WIDTH,
     headerswidth: int = DEFAULT_COLUMN_WIDTH,
     non_unique_behavior: str = "warn",
-    validate: bool = True,  # noqa: FBT001, FBT002
+    validate: str = "madx",
 ) -> None:
     """
-    Writes the provided ``DataFrame`` to disk at **tfs_file_path**, eventually with the `headers_dict` as
-    headers dictionary.
+    Writes the provided `DataFrame` to disk at **tfs_file_path**, eventually with
```

Comment:
I don't understand... Why eventually?

Reply:
The eventually refers to the `headers_dict` argument. If it's provided it will be used as headers, otherwise we use the headers of the frame if it has any, otherwise no headers. It's possible to write from a pandas dataframe and provide headers_dict, instead of constructing a `TfsDataFrame` first and writing. I'll rephrase a bit.